### PR TITLE
Copyedit README.md to keep improving readability

### PR DESCRIPTION
**Pull request by Lupie**
- Should the module in the example be named "rpc" or "yarpc"?
- I changed go.uber.org/config to go.uber.org/fx/config (right or wrong?)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,33 +21,33 @@ To get a feel for what an UberFx service looks like, see our
...
 
-## Core packages
+## Core Packages
 
-The top-level packages (except for `service`) contain the nuts and bolts useful
-to have in a fully-fledged service, but are not specific to an instance of a
-service or even the idea of a service.
+The top-level packages contain the nuts and bolts useful for a fully-fledged
+service. That said, none (except for `service`) requires an instance or even
+the idea of a service. You can use the packages independently their logic.
```
Comment (anuptalwalkar) [inline README.md]: Suggestion: You can use the packages independently _for their value_?.

File: README.md [hunk 2]
```diff
@@ -21,33 +21,33 @@ To get a feel for what an UberFx service looks like, see our
...
 
-If, for example, you just want use the configuration logic from UberFx, you
-could import `go.uber.org/config` and use it in a stand-alone CLI app.
+If, for example, you only want use the configuration logic from UberFx, you
+can import `go.uber.org/fx/config` and use it in a standalone CLI app.
```
Comment (glibsm) [inline README.md]: good catch!

**Comment by sectioneight** [comment]
Can you also run `make gendoc` to update the auto-generated `doc.go` file?

**Comment by sectioneight** [comment]
And I like calling the module "rpc", the fact that it's YARPC is an implementation detail

File: README.md (hunk 1 above)
Comment (Lupie) [inline README.md]: Value sounds like a number, but good catch. Is "for their functionality" too clunky? Or just end the sentence after "independently"?

Comment (anuptalwalkar) [inline README.md]: Either way is fine. "You can use the packages independently" does sound short and precise.

**Pull request merged**